app/javascript/dashboard/store/modules/conversations/helpers.js

```diff
@@ -4,3 +4,34 @@ export const findPendingMessageIndex = (chat, message) => {
     m => m.id === message.id || m.id === tempMessageId
   );
 };
+
+export const applyPageFilters = (chat, filters) => {
```

Comment:
please use conversation instead of chat as it becomes confusing all the time. Let stick to conversation not chat